tests/test_config.py

```diff
@@ -285,3 +285,25 @@ def test_always_use_configured_logger():
     c.set('statsd_host', 'localhost:12345')
     # still uses custom logger over statsd
     assert c.logger_class == MyLogger
+
+def test_load_enviroment_variables_config():
+    os.environ["GUNICORN_WORKERS"] = "4"
```

Comment:
We need to unset these environment variables. I don't know what's the best practice to do this in pytest though (I'd suggest writing a context manager if we were using unittest -- perhaps you need a fixture?)